exporter/tanzuobservabilityexporter/metrics.go

```diff
@@ -115,19 +140,100 @@ type flushCloser interface {
 	Close()
 }
 
+// counter represents an internal counter metric. The zero value is ready to use
+type counter struct {
+	lock  sync.Mutex
+	count int64
```

Comment:
Thank you for pointing this out. I changed to use atomic.AddInt64() and atomic.LoadInt64().  Now my counter struct has a single int64 field in it. I thought about making counter be an int64 instead of a struct, but decided against it because of the type conversions I'd have to do when calling atomic.AddInt64 and atomic.LoadInt64.